src/queries/featureGates/featureConstants.ts

```diff
@@ -42,6 +42,7 @@ export const GCP_WIF_DEFAULT = 'ocmui-gcp-wif-default';
 export const MP_ADDITIONAL_MAINTENANCE_VALUES = 'ocmui-mp-additional-maintenance-values';
 export const WINDOWS_LICENSE_INCLUDED = 'ocmui-windows-license-included';
 export const ALLOW_EUS_CHANNEL = 'ocmui-allow-eus-channel';
+export const PLATFORM_LIGHTSPEED_REBRAND = 'platform.lightspeed-rebrand';
```

Comment:
Hi @LiorKGOW Usually, feature flags follow the prefix pattern "ocmui.*", so it's unclear why this one (platform.lightspeed-rebrand) deviates from that convention. Could you please confirm if this naming is intentional or if it should be aligned with the existing "ocmui" standard?